src/components/PopUpWithForm.js

```diff
@@ -32,5 +51,11 @@ export default class PopUpWithForm extends PopUp {
     this.inputValues.forEach((input) => {
       input.value = data[input.name];
     });
+
+    close();
+    {
+      this._popup.querySelector(".modal__form").reset();
```

Comment:
You haven't corrected my remark https://github.com/mowooje/se_project_aroundtheus/pull/4/files#r1759990759